python/snewpy/_model_downloader.py

```diff
@@ -113,7 +113,7 @@ def from_zenodo(zenodo_id:str, model:str, filename:str, path:str=model_path):
 
     Returns
     -------
-    file : FileHandle object.
+    file_url, md5sum
     """
     zenodo_url = f'https://zenodo.org/api/records/{zenodo_id}'
     path = Path(path)/str(model)
```

Comment:
Cleanup: Can you remove the remaining references to `path` in the docstring and function body?

Reply:
(and same in the `from_github` docstring)

Reply:
done